src/test/java/com/scalefocus/blog_api/service/impl/BlogServiceImplTest.java

```diff
@@ -0,0 +1,64 @@
+package com.scalefocus.blog_api.service.impl;
+
+import com.scalefocus.blog_api.dto.BlogDto;
+import com.scalefocus.blog_api.entity.Blog;
+import com.scalefocus.blog_api.mapper.BlogMapper;
+import com.scalefocus.blog_api.repository.BlogRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.co.jemos.podam.api.PodamFactory;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+
+@ExtendWith(MockitoExtension.class)
+public class BlogServiceImplTest {
+
+    @Mock
+    private BlogRepository blogRepository;
+
+    @Mock
+    private BlogMapper blogMapper;
+
+    @InjectMocks
+    private BlogServiceImpl blogServiceImpl;
+
+    private Blog blog;
+    private BlogDto blogDto;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+        PodamFactory podamFactory = new PodamFactoryImpl();
```

Comment:
Try to use such libraries for more complex object creations. For easy to create objects use the normal way. 
Also consider downsides for PodamFactory before using it in real project, such as _" If a test fails due to random data, it might be harder to debug since the exact input that caused the failure might not be immediately obvious. You may need to log or capture the generated data to understand what went wrong."_